src/contracts/account/ProxyAccountDeployer.sol

```diff
@@ -91,7 +91,7 @@ contract ProxyAccountDeployer {
      * Creates base Account for contracts
      */
     constructor() public {
-        baseAccount = address(new ProxyAccount());
+        baseAccount = address(new ProxyAccount{ salt : "v0.1.0" }());
```

Comment:
this is funky syntax. what is it doing actually?

Reply:
Creating the proxy account using create2 and passing the salt

Reply:
so weird lol